socialhome/streams/app/main.js

```diff
@@ -0,0 +1,11 @@
+import Vue from "vue"
+import VueResource from "vue-resource"
+import "./stores/streamStore"
+import "./components/Stream.vue"
+
+Vue.use(VueResource)
+
+export default new Vue({
+    el: "#app",
+    http: {root: "/"}
```

Comment:
This doesn't seem right. Only one user set stream is at `/` - mostly streams are just everywhere. What is this for?